### PR TITLE
Fix fullscreen event handling in Firefox.

### PR DESCRIPTION
**Pull request by toji**
Allows distortion to work on that platform.

### CODE REVIEW

File: src/base.js [hunk 1]
```diff
@@ -225,14 +225,14 @@ VRDisplay.prototype.addFullscreenListeners_ = function(element, changeHandler, e
   if (changeHandler) {
     element.addEventListener('fullscreenchange', changeHandler, false);
     element.addEventListener('webkitfullscreenchange', changeHandler, false);
-    element.addEventListener('mozfullscreenchange', changeHandler, false);
+    document.addEventListener('mozfullscreenchange', changeHandler, false);
```
Comment (cvan) [inline src/base.js]: be sure to do https://github.com/borismus/webvr-polyfill/pull/39/files#diff-d7e697eff3913f1acaacac8002c3b05eR250

File: src/base.js [hunk 2]
```diff
@@ -225,14 +225,14 @@ VRDisplay.prototype.addFullscreenListeners_ = function(element, changeHandler, e
...
     element.addEventListener('msfullscreenchange', changeHandler, false);
   }
 
   if (errorHandler) {
     element.addEventListener('fullscreenerror', errorHandler, false);
     element.addEventListener('webkitfullscreenerror', errorHandler, false);
-    element.addEventListener('mozfullscreenerror', errorHandler, false);
+    document.addEventListener('mozfullscreenerror', errorHandler, false);
```
Comment (cvan) [inline src/base.js]: and https://github.com/borismus/webvr-polyfill/pull/39/files#diff-d7e697eff3913f1acaacac8002c3b05eR258

**Comment by toji** [comment]
Thanks for catching those, @cvan! Fixed! (And generated the build file this time around too.)

**Comment by borismus** [comment]
Thanks!

**Pull request merged**